File: 1.Two Sum/Java/Solution-2016-01-12.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] index = new int[2];
        int len = nums.length;
        for(int i=0; i<len; i++) {
            for(int j=i+1; j<len; j++) {
                if(nums[i] + nums[j] == target) {
                    index[0] = i+1;
                    index[1] = j+1;
                    return index;
                }
            }
        }
        return index;
    }
}

File: 1.Two Sum/C++/Solution-2016-01-12.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> index;
        int len = nums.size();
        for(int i=0; i<len; i++) {
            for(int j=i+1; j<len; j++) {
                if((nums[i] + nums[j]) == target) {
                    index.push_back(i+1);
                    index.push_back(j+1);
                    return index;
                }
            }
        }
    }
};